account_payment_partner/models/account_move.py

```diff
@@ -65,6 +66,18 @@ def _compute_payment_mode(self):
                     move.payment_mode_id = move.with_context(
                         force_company=move.company_id.id
                     ).partner_id.customer_payment_mode_id
+                elif (
+                    move.type in ["out_refund", "in_refund"] and move.reversed_entry_id
+                ):
+                    move.payment_mode_id = move.reversed_entry_id.payment_mode_id
```

Comment:
This is not correct. We have to take the payment mode for the refunds of the reversed entry payment mode. Anyways, I'm not sure if we have to do this one here, as this is only intended for manual refunds.

Reply:
Exactly, these code is related to manual refunds. 
Exists too in `_reverse_move_vals()` function `payment_mode_id` to keep previous value.

Reply:
But that's incorrect and the reason why we are doing all these changes, for avoiding a payment mode whose money direction is the contrary. The comment in the code saying that it's better to keep the same payment mode is also incorrect.

Reply:
OK, i have changed to `refund_payment_mode_id`. Is correct now right?